src/models/participantEndpoint/facade.js

```diff
@@ -29,8 +29,10 @@ const Logger = require('@mojaloop/central-services-logger')
 const Util = require('@mojaloop/central-services-shared').Util
 const Enums = require('@mojaloop/central-services-shared').Enum
 const ErrorHandler = require('@mojaloop/central-services-error-handling')
+const JwsSigner = require('@mojaloop/sdk-standard-components').Jws.signer
 const Mustache = require('mustache')
 const Config = require('../../lib/config')
+const uriRegex = /(?:^.*)(\/(participants|parties|quotes|transfers)(\/.*)*)$/
```

Comment:
Is this a magic string that will remain static or we should make it a configurable variable having that as the default value? If is likely to stay static, why put it on the top scope? If it has to stay in the top scope, I would suggest to name it in capitals, like `URI_REGEX`, which is the convention for such things.

Reply:
It is not a string @vbarzokas it is an instance of a regex.

Reply:
Yes correct, thanks. The rest suggestion remains the same though.